### ISSUE TITLE
How to arrive at one's destination and return to one's original position?

### ISSUE THREAD

**Issue by ghwns0529** [robotemi/sdk#284]
How to arrive at one's destination and return to one's original position?

**Comment by lcgao** [comment]
Hi, you can use the Auto return in Settings -> General Settings -> Auto Return to select a location as the original location and temi will back to the original location in an exactly duration.

**Comment by ghwns0529** [comment]
Thank you!! If I programming with Temi SDK,  should I use autoReturnOn method?

**Comment by lcgao** [comment]
Yes you can use autoReturnOn method to turn on the Auto return but can't set the duration and location. Please note that the required permission for the method.
